src/page-sentencing/PageSentencing.js

```diff
@@ -4,20 +4,44 @@ import useChartData from "../hooks/useChartData";
 import VizSentenceTypes from "../viz-sentence-types";
 import VizSentenceLengths from "../viz-sentence-lengths";
 
+import VizSentencePopulation from "../viz-sentence-population";
+
 export default function PageSentencing() {
   const { apiData, isLoading } = useChartData("us_nd/sentencing");
 
   if (isLoading) {
     return null;
   }
 
+  const sentenceTypeByDistrictByDemographics =
+    apiData.sentence_type_by_district_by_demographics;
+  const populatedDistricts = new Set(
+    sentenceTypeByDistrictByDemographics.map((record) => record.district)
+  );
+  const districtIsPopulated = (record) => populatedDistricts.has(record.id);
+
   const TITLE = "Sentencing";
   const DESCRIPTION = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
     Vestibulum in finibus tellus, et ullamcorper augue. Quisque eleifend
     tortor vitae iaculis egestas. Donec dictum, nunc nec tincidunt cursus,
     ipsum dui gravida.`;
 
   const SECTIONS = [
+    {
+      title: "Who is being sentenced?",
+      description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Vestibulum in finibus tellus, et ullamcorper augue. Quisque eleifend
+      tortor vitae iaculis egestas. Donec dictum, nunc nec tincidunt cursus,
+      ipsum dui gravida.`,
+      showLocationControl: true,
+      locationControlLabel: "Judicial District",
+      VizComponent: VizSentencePopulation,
+      vizData: {
+        populationDemographics: sentenceTypeByDistrictByDemographics,
```

Comment:
I don't have access to a separate sentencing population file, but from what I can tell the data returned from `apiData.sentence_type_by_district_by_demographics` should be reusable for this use case.